consensus/types/src/subnet_id.rs

```diff
@@ -78,45 +77,15 @@ impl SubnetId {
     /// Computes the set of subnets the node should be subscribed to during the current epoch,
     /// along with the first epoch in which these subscriptions are no longer valid.
     #[allow(clippy::arithmetic_side_effects)]
-    pub fn compute_subnets_for_epoch<E: EthSpec>(
+    pub fn compute_attestation_subnets<E: EthSpec>(
         node_id: ethereum_types::U256,
-        epoch: Epoch,
         spec: &ChainSpec,
-    ) -> Result<(impl Iterator<Item = SubnetId>, Epoch), &'static str> {
-        // simplify variable naming
-        let subscription_duration = spec.epochs_per_subnet_subscription;
+    ) -> impl Iterator<Item = SubnetId> {
+        // The bits of the node-id we are using to define the subnets.
         let prefix_bits = spec.attestation_subnet_prefix_bits as u64;
```

Comment:
I think your spec PR suggests replacing this with `int(ceillog2(ATTESTATION_SUBNET_COUNT)`?

Reply:
Yeah. I've left it as a config value. The default in the config is 6, which is the value for 64 subnets. 

If someone changes the ATTESTATION_SUBNET_COUNT in the config, they SHOULD then update the attestation_subnet_prefix to match this. 

The reason I left it as a config, is that then its stored on the spec and we don't have to comptuer the int(ceillog2()) each time, i.e it becomes static. 

Also, its not the end of the world if someone wants to configure this out of spec.

Reply:
I see, make sense!